Pod/API/PublicRemoteMethods/consoleCommand.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  consoleCommand.swift
+//  ElJayKit
+//
+//  Created by Isaac Greenspan on 12/14/15.
+//  Copyright © 2015 Isaac Greenspan. All rights reserved.
+//
+
+import Foundation
+import AlamofireXMLRPC
+
+extension API {
+    public enum ConsoleCommandOutputLineType: String {
+        case Normal = ""
+        case Info = "info"
+        case Error = "error"
+        case Success = "success"
+    }
+    
+    public struct ConsoleCommandOutputLine {
+        let type: ConsoleCommandOutputLineType
+        let text: String
+        
+        var display: String {
+            return NSString(format: "[%-7s] %@", (self.type.rawValue as NSString).UTF8String, self.text) as String
```

Comment:
The `%-7s` bit didn't work right in `String(format: ...)`.  I don't remember exactly how not right, just that it wasn't right and that StackOverflow confirmed that `String(format: ...)` doesn't support the full set of string formatting stuff that `NSString` does.